src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  box-sizing: border-box;
+  max-width: $maxCalendarWidth;
+  padding: $paddings;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  @each $day, $margin in $days-of-week {
+    &--start-day-#{$day} .calendar__day:first-child {
+      margin-left: ($cellWidth + 1) * ($margin - 1);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    background: #eee;
+    width: 100px;
+    height: 100px;
+
+    border: 1px solid #000;
+
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    transition: all 500ms;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        color: #000;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```

Comment:
```suggestion

```
Consider moving these styles to a basic class instead of generating them for each element in the cycle

Reply:
In my opinion I should move only font properties. Am I right?
 font-family: Arial, sans-serif;
 font-size: 30px;
 color: #000;